src/cmd/run.js

```diff
@@ -323,7 +328,16 @@ export class ExtensionRunner {
   }
 
   run(profile: FirefoxProfile): Promise<FirefoxProcess> {
-    const {firefoxApp, firefox} = this;
-    return firefoxApp.run(profile, {firefoxBinary: firefox});
+    const binaryArgs = [];
+    const {firefoxApp, firefox, startUrl} = this;
+    if (startUrl) {
+      let urls = Array.isArray(startUrl) ? startUrl : [startUrl];
```

Comment:
let -> const (this raise an eslint error on travis: https://travis-ci.org/mozilla/web-ext/jobs/188307450#L1217)